### ISSUE TITLE
Runs downtown on php 8 ?

### ISSUE THREAD

**Issue by hcarsten** [shopwareDowntown/downtown#370]
is it possible to run the system on php 8 ?

**Comment by shyim** [comment]
Shopware 6.4 supports 8.0

**Comment by hcarsten** [comment]
yes I know but in the composer.json here (in downtown) are still references to SW 6.3 and also the  PHP dependency is on PHP ~7.2
